gremlin/templates/gremlin-service-account.yaml

```diff
@@ -5,6 +5,10 @@ kind: ServiceAccount
 metadata:
   name: gremlin
   namespace: {{ .Release.Namespace }}
+{{ if .Values.gremlin.serviceAccount.iamRole }}
+  annotations:
+    eks.amazonaws.com/role-arn: {{ .Values.gremlin.serviceAccount.iamRole }}
+{{ end }}
```

Comment:
By doing this, we're locking users into the `eks.amazonaws.com/role-arn` annotation being the only annotation supported. I'd prefer we go with the https://github.com/gremlin/helm/pull/88/files#diff-ee3b9fdbccfe887c3de3569decd527ad5f930f6f9c76b2c1086fd11f113ed9b3R9-R12 paradigm of allowing them to set any annotations they want.

1) It opens the door for other potential use cases of annotating service accounts
2) It means we won't have to make chart changes if we choose to support Google Cloud KMS as well.

Reply:
Allowed both setting annotations and iamRole to live in annotations